ui/raidboss/emulator/data/network_log_converter/LineEvent0x19.ts

```diff
@@ -0,0 +1,63 @@
+import LineEvent from './LineEvent';
+import EmulatorCommon from '../../EmulatorCommon';
+import LogRepository from './LogRepository';
+
+// Combatant defeated event
+export class LineEvent0x19 extends LineEvent {
+  public resolvedName: string | false;
+  public resolvedTargetName: string | false;
```

Comment:
```suggestion
  public resolvedName?: string;
  public resolvedTargetName?: string;
```

What about undefined instead of false as an unset type?